android/src/test/java/com/reactnativecommunity/asyncstorage/next/TestUtils.kt

```diff
@@ -1,84 +1,19 @@
 package com.reactnativecommunity.asyncstorage.next
 
-import com.facebook.react.bridge.Dynamic
+import com.facebook.react.bridge.JavaOnlyArray
 import com.facebook.react.bridge.ReadableArray
-import com.facebook.react.bridge.ReadableMap
-import com.facebook.react.bridge.ReadableType
-import com.facebook.react.bridge.WritableArray
 
-fun <T : Any> createNativeCallArguments(vararg list: T): ReadableArray {
-    val args = ArrayList(list.toList())
-    return TestingArguments(args)
+fun List<Any?>.toReadableArray(): ReadableArray {
```

Comment:
Nice one 👍 
You could actually move this extension to `ArgumentHelpersTest.kt` file, to remove this one

Reply:
Yeah, I think that makes sense.